### PR TITLE
Add Cynosa Chroma support

### PR DESCRIPTION
**Pull request by xf-**
Ordered the supported ornata chroma and not supported cynosa chroma (not pro) and both are very similar. Code works for me with razergenie, but didn't know how to find the assets (images).

### CODE REVIEW

**Comment by z3ntu** [comment]
Does everything work?

**Comment by z3ntu** [comment]
And please fix the CI failure by running `autopep8 --recursive --in-place --max-line-length 500 daemon/` .

**Comment by xf-** [comment]
I tested all animations - one feature i dind't find is Fn Backlight. If i press FN - FN has another color.

**Comment by xf-** [comment]
@z3ntu is new error:  r,device_serial,XX0000000034 a random number?

**Comment by z3ntu** [comment]
You can use `./scripts/get_next_fake_driver_serial.sh` and use the serial outputted there.

File: README.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ Also this is the master branch, devices may appear in the tables below but might
 | Razer Blade Stealth (Late 2016)               |  1532:0220  |
 | Razer BlackWidow Chroma V2                    |  1532:0221  |
 | Razer Blade (Late 2016)                       |  1532:0224  |
+| Razer Cynosa                                  |  1532:022A  |
```
Comment (z3ntu) [inline README.md]: `Razer Cynosa Chroma` here

File: daemon/openrazer_daemon/hardware/keyboards.py [hunk 2]
```diff
@@ -629,6 +629,31 @@ class RazerOrnataChroma(_RippleKeyboard):
...
+               'set_reactive_effect', 'set_none_effect', 'set_breath_random_effect', 'set_breath_single_effect', 'set_breath_dual_effect',
+               'set_custom_effect', 'set_key_row', 'get_game_mode', 'set_game_mode', 'get_macro_mode', 'set_macro_mode',
+               'get_macro_effect', 'set_macro_effect', 'get_macros', 'delete_macro', 'add_macro',
+               'set_starlight_random_effect', 'set_starlight_single_effect', 'set_starlight_dual_effect',
+               'set_ripple_effect', 'set_ripple_effect_random_colour']
+
+    RAZER_URLS = {
+        "top_img": "https://assets.razerzone.com/eeimages/products/25713/razer-ornata-chroma-gallery-05.png",
+        "side_img": "https://assets.razerzone.com/eeimages/products/25713/razer-ornata-chroma-gallery-07.png",
+        "perspective_img": "https://assets.razerzone.com/eeimages/products/25713/razer-ornata-chroma-gallery-08.png"
```
Comment (z3ntu) [inline daemon/openrazer_daemon/hardware/keyboards.py]: as you said, images have to be replaced

File: driver/razerkbd_driver.c [hunk 3]
```diff
@@ -190,6 +190,7 @@ void razer_set_device_mode(struct usb_device *usb_dev, unsigned char mode, unsig
     switch(usb_dev->descriptor.idProduct) {
     case USB_DEVICE_ID_RAZER_ORNATA:
     case USB_DEVICE_ID_RAZER_ORNATA_CHROMA:
+    case USB_DEVICE_ID_RAZER_CYNOSA_CHROMA:
```
Comment (z3ntu) [inline driver/razerkbd_driver.c:210]: Does it also work without this line? If yes, remove it

**Review by z3ntu — CHANGES_REQUESTED** [review]
see comments

File: daemon/openrazer_daemon/hardware/keyboards.py (hunk 2 above)
Comment (z3ntu) [reply daemon/openrazer_daemon/hardware/keyboards.py]: I found https://assets.razerzone.com/eeimages/support/products/1256/1256_cynosa_chroma.png that you could use as top_img and remove the others until we have some.

File: driver/razerkbd_driver.c (hunk 3 above)
Comment (xf-) [reply driver/razerkbd_driver.c:210]: (strange - github showed me only one comment and after a force push the next).
Wave, none, Spectrum and brightness after i remove those lines.

Comment (z3ntu) [reply driver/razerkbd_driver.c:210]: So does it work or does it not?
And please don't force-push all the time, that way I have to look over the complete patch every time. The commits can be squashed before merging.

Comment (xf-) [reply driver/razerkbd_driver.c:210]: It won't work with deleted cases.

File: driver/razerkbd_driver.h [hunk 4]
```diff
@@ -110,6 +110,10 @@
 #define USB_DEVICE_ID_RAZER_ORNATA_CHROMA 0x021e
 #endif
 
+#ifndef USB_DEVICE_ID_RAZER_CYNOSA_CHROMA
+#define USB_DEVICE_ID_RAZER_CYNOSA_CHROMA 0x022a
+#endif
```
Comment (z3ntu) [inline driver/razerkbd_driver.h]: please put this block where it fits sorted by the number (so after USB_DEVICE_ID_RAZER_BLADE_LATE_2016)

File: driver/razerkbd_driver.c (hunk 3 above)
Comment (z3ntu) [reply driver/razerkbd_driver.c:210]: Okay if it doesn't work without, leave it in.

**Comment by terricain** [comment]
:+1:

File: driver/razerkbd_driver.h (hunk 4 above)
Comment (xf-) [reply driver/razerkbd_driver.h]: OK - i hope the next days. Really busy and i don't have always access to keyboard.

Comment (Vehyla) [reply driver/razerkbd_driver.h]: Could we add the razer cynosa chroma pro with 0x022c?

Comment (z3ntu) [reply driver/razerkbd_driver.h]: @Vehyla No, the devices are not the same.

Comment (xf-) [reply driver/razerkbd_driver.h]: finally moved (characters are mixed lower- and uppercase - currently lower)

**Comment by xf-** [comment]
rebased on current master and  changed to uppercase char in hex

**Review by z3ntu — APPROVED** [review]
LGTM

**Comment by pegahcarter** [comment]
Can we add this for the pro version, too??  
1532:022c

**Comment by z3ntu** [comment]
@carlfarterson See #464

**Comment by xf-** [comment]
I only own the normal version. I think i also included a small bug. After suspend the Keyboard is dark and i need to select the mode in RazerGenie again. I will debug it someday.

**Pull request merged**